connector/connect/client/jvm/pom.xml

```diff
@@ -136,4 +144,37 @@
       </plugin>
     </plugins>
   </build>
+
+  <!-- To skip client integration test, use 'mvn clean package -DskipJvmClientITs=true' -->
```

Comment:
No need to add this profile, We can use `-Dtest.exclude.tags=org.apache.spark.sql.connect.client.util.ClientIntegrationTest` directly.